src/Parser/GeoJsonParser.js

```diff
@@ -144,6 +144,12 @@ const keyProperties = ['type', 'geometry', 'properties'];
 const firstCoordinates = a => (a === undefined || (Array.isArray(a) && !isNaN(a[0])) ? a : firstCoordinates(a[0]));
 
 function jsonFeatureToFeature(crsIn, json, collection) {
+    if (!json.geometry || !json.geometry.type) {
+        console.warn('No geometry provided');
+        // throw new Error(`No geometry provided ${json}`);
```

Comment:
Could you simplify with `if (!json.geometry?.type)` and remove comment on line 149 ?